nixos/modules/services/networking/ntopng.nix

```diff
@@ -64,6 +72,23 @@ in
         '';
       };
 
+      redis.address = mkOption {
+        type = types.str;
+        example = literalExpression "config.services.redis.ntopng.unixSocket";
+        description = ''
+          Redis address - may be a Unix socket or a network host and port.
+        '';
+      };
+
+      redis.createInstance = mkOption {
+        type = types.nullOr types.str;
```

Comment:
Missing default value?

Reply:
It has one [here](https://github.com/NixOS/nixpkgs/pull/159538/commits/249c38532449ebcb8c5d3f814049f98cad3cd190#diff-89bb9a9988f12cbf17c0ea5dd3e856e5b347ff33913ec9820878e93bc9f5aa2cR125), but it depends on `system.stateVersion`. Not sure how to best express dynamic defaults like this -- we can also put it into `default` here, but it will always be equal to `"ntopng"` in documentation. What do you think?

Reply:
I have no strong opinion.

Reply:
After some thought, I changed this to have `if` in `default`, with a note in description that old `stateVersion` uses a different default. Seems cleaner.